deploy.yaml

```diff
@@ -0,0 +1,79 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: secrethub-mutating-webhook
+  labels:
+    app: secrethub-mutating-webhook
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: secrethub-mutating-webhook
+  template:
+    metadata:
+      labels:
+        app: secrethub-mutating-webhook
+    spec:
+      containers:
+      - name: secrethub-mutating-webhook
+        image: simonbarendse/kubernetes-mutating-webhook:latest
+        volumeMounts:
+        - name: webhook-certs
+          mountPath: /etc/webhook/certs
+          readOnly: true
+      volumes:
+      - name: webhook-certs
+      initContainers:
+        - name: secrethub
+          image: secrethub/cli:0.38.0
+          env:
+            - name: SECRETHUB_CREDENTIAL
+              valueFrom:
+                secretKeyRef:
+                  name: secrethub-mutating-webhook
+                  key: secrethub_credential
```

Comment:
This can be replaced with native authentication in the case of AWS EKS and soon also with native authentication on GKE.